src/store/pattern/pattern.ts

```diff
@@ -15,219 +19,77 @@ import { TypeScriptParser } from './parser/typescript-parser';
  * e.g. Patternplate.
  */
 export class Pattern {
-	/**
-	 * The parent folder containing the pattern folder.
-	 */
-	protected folder: PatternFolder;
-
-	/**
-	 * The ID of the pattern (also the folder name within the parent folder).
```

Comment:
Please re-add the documentation that is still valid.
Probably still valid:
- ID
- iconPath
- constructor

Reply:
done ✅